packages/frameworks-sveltekit/src/lib/actions.ts

```diff
@@ -78,11 +82,11 @@ export async function signOut(
   config: SvelteKitAuthConfig,
   event: RequestEvent
 ) {
-  const { request } = event
+  const { request, url: { protocol } } = event
   const headers = new Headers(request.headers)
   headers.set("Content-Type", "application/x-www-form-urlencoded")
 
-  const url = createActionURL("signout", headers, config.basePath)
+  const url = createActionURL("signout", protocol, headers, process.env, config.basePath)
```

Comment:
for SvelteKit we should use the env object from [`"$env/dynamic/private"`](https://kit.svelte.dev/docs/modules#$env-dynamic-private) - that is SvelteKit's abstraction to access env vars from different adapters
```suggestion
  const url = createActionURL("signout", protocol, headers,env, config.basePath)
```